specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -7,7 +7,7 @@
   },
   "paths": {},
   "definitions": {
-    "ChatMessageReceivedEventData": {
```

Comment:
This is a breaking change in a stable swagger.  Why is this required?

Reply:
I created this swagger on friday, missed adding the prefix as the guidelines. So wanted update it.

Reply:
The service is yet to launch in public preview, it will launch next month.

Reply:
Then I don't think this swagger file should be in a stable directory yet - it sends the wrong message to customers.